src/Orchard.Cms.Web/Modules/Orchard.Tenants/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "2.0.0-*",
+  "dependencies": {
+    "Microsoft.Extensions.DependencyInjection": "1.0.0",
+    "Orchard.Environment.Navigation": "2.0.0-*",
+    "Orchard.Admin.Abstractions": "2.0.0-*",
+    "Orchard.Hosting": "2.0.0-*",
+    "Orchard.DisplayManagement": "2.0.0-*",
+    "Orchard.Recipes": "2.0.0-*"
```

Comment:
DisplayManagement is necessary for the base razor page. 
I could remove Recipes.